onlineapi/src/main/kotlin/com/codingblocks/onlineapi/api/OnlineJsonApi.kt

```diff
@@ -187,6 +187,17 @@ interface OnlineJsonApi {
         @Query("sort") sort: String = "-postedOn"
     ): Call<ArrayList<Jobs>>
 
+    @GET("jobs")
```

Comment:
no need to make a separate method  for this update the existing method and use that only

Reply:
updated @aggarwalpulkit596